CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## UNRELEASED
+
+- Adds a selection for adding a `robots.txt` string to the site through the global configuration.
```

Comment:
I would say robots.txt "file", maybe even with the literal quotes to acknowledge that it's generated virtually for those who know, but everybody says "robots.txt file"

Reply:
Changed.